libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -460,19 +460,19 @@ void AC_PosControl::init_takeoff()
 // is_active_z - returns true if the z-axis position controller has been run very recently
 bool AC_PosControl::is_active_z() const
 {
-    return ((AP_HAL::millis() - _last_update_z_ms) <= POSCONTROL_ACTIVE_TIMEOUT_MS);
+    return ((AP_HAL::micros64() - _last_update_z_us) <= POSCONTROL_ACTIVE_TIMEOUT_US);
```

Comment:
is there a AP_HAL::micros32() function call? if yes, woudn't it be enough to use that here ?

Reply:
@amilcarlucas, thanks for taking a look.  there doesn't appear to be a micros32() anywhere..

Reply:
`AP_HAL::micros()`

However, you may wish to consider the rollover time of 32-bit microseconds is somewhere over an hour, which is within a runtime of our vehicles.